README.md

```diff
@@ -0,0 +1,92 @@
+# Infection Static Analysis Plugin
+
+This plugin is designed to run static analysis on top of [`infection/infection`](https://github.com/infection/infection)
+test runs in order to discover if [escaped mutants](https://en.wikipedia.org/wiki/Mutation_testing)
+are valid mutations, or if they do not respect the type signature of your
+program.
+
+## Usage
+
```

Comment:
The current design of this tool is to run `vendor/bin/roave-infection-static-analysis-plugin` instead of running `vendor/bin/infection`.

Reply:
b7b501460592ecacbce781c8dc0e383917dd06fa